README.md

```diff
@@ -149,4 +149,88 @@ As reviewers, a PR should be only approved if following conditions are met (In o
 - git add \<file or folder\>
 - git commit -m \<meaningful commit message\>
 - git push
-- git pull
\ No newline at end of file
+- git pull
+
+## Database connection
+1. Run the npm install
+2. configure the service account (in first iterations we will use a default credetial)
```

Comment:
Mateo here we can maybe explain better how to use the credential, tomorrow we can do it

Reply:
yeah but for now we don't need it, then we will add it tomorrow